tools/azsdk-cli/Azure.Sdk.Tools.Cli/Templates/ReadMeGenerator/README-template.go.md

```diff
@@ -0,0 +1,106 @@
+# Azure Template Package client library for Go
+
+<Describe the service, and what kinds of features it provides - this should be a short paragraph>
+
+Use the <package> client module `github.com/Azure/azure-sdk-for-go/sdk/<package path>` in your application to:
+
+<give a bullet point list of activities you can do with this client>
+
+Key links:
+
+[Source code][source] | [API reference documentation][docs] | [Product documentation][product_docs] | [Samples][go_samples]
+
+## Getting started
+
+### Prerequisites
+
+- [Supported](https://aka.ms/azsdk/go/supported-versions) version of Go - [Install Go](https://go.dev/doc/install)
+- Azure subscription - [Create a free account](https://azure.microsoft.com/free/)
+- <Include examples on how to deploy this resource using the Azure CLI, and via the Azure Portal>
+
+### Install the package
+
+Install the Azure <TODO: package name goes here> client module for Go with [go get][goget]:
```

Comment:
```suggestion
Install the Azure <TODO: package name goes here> client module for Go with [go get](goget):
```

Reply:
I just removed the link. The command example is right below it, and there's literally nothing for the user to install.